src/services/item/plugins/itemCategory/schemas.ts

```diff
@@ -119,12 +93,13 @@ export const getByCategories = {
     type: 'object',
     required: ['categoryId'],
     properties: {
-      categoryId: {
-        type: 'array',
-        items: {
-          $ref: 'https://graasp.org/categories/#/definitions/concatenatedIds',
-        },
-      },
+      categoryId: Type.Array(
+        Type.String({
+          pattern:
+            '^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}' +
+            '(,[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12})*$',
```

Comment:
It is possible to reuse the UUID type `[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}` to avoid duplications? I found 10 results for this pattern, so maybe it could be interesting to have the possibility to reuse it?

```suggestion
            `^${UUID_PATTERN}(,${UUID_PATTERN})*$`,
```